pyporscheconnectapi/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
pyporscheconnectapi/cli.py,sha256=Vb6kehjJ2W1QbQKfvox8-X5-YPVRTFpgBJiw4M_nDAg,10781
pyporscheconnectapi/client.py,sha256=9n4dQM_GJOxkCtRaFZGLAP13F_5g2ylIjY4R_vUTRqs,18192
pyporscheconnectapi/connection.py,sha256=2RyyngTnv-WTZW7KSwQ_9BEnm2Y2uZ3x9erlIQBwNLs,10073
pyporscheconnectapi/exceptions.py,sha256=Ez_D_TsafBd59LRIKRvGFv_6W9Ia425QCaClwXEke2w,1539
pyporscheconnectapi-0.0.17.dist-info/LICENSE,sha256=3wo4ylCH88NDNuSoNFaeaNCdzpQDloqsIUDQMg5FWec,1071
pyporscheconnectapi-0.0.17.dist-info/METADATA,sha256=xJ9drOhtXAHZ-ytpZJ9USePr4QdHaYJ82_efOKG-N9Q,512
pyporscheconnectapi-0.0.17.dist-info/WHEEL,sha256=ewwEueio1C2XeHTvT17n8dZUJgOvyCWCt0WVNLClP9o,92
pyporscheconnectapi-0.0.17.dist-info/entry_points.txt,sha256=cq4djPuFhhopjtdIGqJ6w6vXOb6g65uLQSHnl7HAjfs,60
pyporscheconnectapi-0.0.17.dist-info/top_level.txt,sha256=YXSR3DO6I6QQ0lEQOvTts5BFIGsfOw_WtdFlD8BN4Yw,20
pyporscheconnectapi-0.0.17.dist-info/RECORD,,
